<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QPacman</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qpacman.qrc">
    <normaloff>:/pics/Pacman-arch_logo.png</normaloff>:/pics/Pacman-arch_logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="PacmanView" name="packetView"/>
          </item>
          <item>
           <widget class="PacmanWaitView" name="waitView"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Info</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="margin">
          <number>1</number>
         </property>
         <item>
          <widget class="PacmanInfoBrowser" name="pacInfoView"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Files</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="margin">
          <number>1</number>
         </property>
         <item>
          <widget class="FilesListWidget" name="filesTree">
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="actionsToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMark_All"/>
   <addaction name="actionReset"/>
   <addaction name="separator"/>
   <addaction name="actionLog"/>
   <addaction name="actionApply"/>
  </widget>
  <widget class="PacmanToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefreshList"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionCacheCleanUp"/>
  </widget>
  <action name="actionApply">
   <property name="icon">
    <iconset resource="qpacman.qrc">
     <normaloff>:/pics/dialog-ok-apply.png</normaloff>:/pics/dialog-ok-apply.png</iconset>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
   <property name="toolTip">
    <string>Applies the marked actions</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="qpacman.qrc">
     <normaloff>:/pics/edit-undo.png</normaloff>:/pics/edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Resets all changes were done</string>
   </property>
   <property name="shortcut">
    <string>Alt+R</string>
   </property>
  </action>
  <action name="actionMark_All">
   <property name="icon">
    <iconset resource="qpacman.qrc">
     <normaloff>:/pics/edit-select-all.png</normaloff>:/pics/edit-select-all.png</iconset>
   </property>
   <property name="text">
    <string>Mark All</string>
   </property>
   <property name="toolTip">
    <string>Marks all items</string>
   </property>
   <property name="shortcut">
    <string>Alt+M</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="qpacman.qrc">
     <normaloff>:/pics/refresh-cache.png</normaloff>:/pics/refresh-cache.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Updates the repository database and refresh the package list</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qpacman.qrc">
     <normaloff>:/pics/kpk-important.png</normaloff>:/pics/kpk-important.png</iconset>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
   <property name="toolTip">
    <string>Shows the messages from the packages' post scripts</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionRefreshList">
   <property name="icon">
    <iconset resource="qpacman.qrc">
     <normaloff>:/pics/view-refresh.png</normaloff>:/pics/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>RefreshList</string>
   </property>
   <property name="toolTip">
    <string>Refresh package list only</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionCacheCleanUp">
   <property name="icon">
    <iconset resource="qpacman.qrc">
     <normaloff>:/pics/package-clean-up.png</normaloff>:/pics/package-clean-up.png</iconset>
   </property>
   <property name="text">
    <string>CacheCleanUp</string>
   </property>
   <property name="toolTip">
    <string>Removes all packages from the pacman's cache</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PacmanInfoBrowser</class>
   <extends>QTextBrowser</extends>
   <header>pacmaninfobrowser.h</header>
  </customwidget>
  <customwidget>
   <class>FilesListWidget</class>
   <extends>QTreeWidget</extends>
   <header>fileslistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>PacmanToolBar</class>
   <extends>QToolBar</extends>
   <header>pacmantoolbar.h</header>
  </customwidget>
  <customwidget>
   <class>PacmanView</class>
   <extends>QTreeView</extends>
   <header>pacmanview.h</header>
  </customwidget>
  <customwidget>
   <class>PacmanWaitView</class>
   <extends>QGraphicsView</extends>
   <header>pacmanwaitview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qpacman.qrc"/>
 </resources>
 <connections/>
</ui>
